src/createRestyleComponent.tsx

```diff
@@ -15,7 +16,7 @@ const createRestyleComponent = <
 ) => {
   const RestyleComponent = (
     props: {
-      style?: any;
+      style?: TStyle;
```

Comment:
Hmm, it seems to me that we could just remove this whole `{ style?: any }` type and just set `props: Props` directly, since the `Props` type should already include the correctly typed style prop within it. Not exactly sure why the `style?: any` part was here in the first place, could you confirm you get the desired effect by removing it altogether?

Reply:
Good catch!  Will update the PR.